### PR TITLE
➕ Install stylelint

### PR DESCRIPTION
**Pull request by fgottesmann**
https://stylelint.io/user-guide/get-started
Setup script+cnfg https://github.com/prettier/stylelint-config-prettier

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["stylelint-config-prettier"]
```
Comment (fre-ben) [inline .stylelintrc.json]: Seems like there's missing "stylelint-config-standard"

**Review by fre-ben — COMMENTED** [review]
GJ!

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
+    "stylelint-check": "stylelint-config-prettier-check",
```
Comment (phlgr) [inline package.json:11]: The stylelint script to check the css files seems to be missing.

**Review by phlgr — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (fgottes) [reply package.json:10]: npm run stylelint-check? I can use that and get the following reply: > fullstack-template@0.1.0 stylelint-check
> stylelint-config-prettier-check

No conflicting rules detected in your stylelint configuration!

**Comment by DNaunin** [comment]
GJ! 🚀

**Pull request merged**